app/components/views/PrivacyPage/PrivacyPage.jsx

```diff
@@ -0,0 +1,50 @@
+import { TabbedPage, TabbedPageTab as Tab, StandaloneHeader } from "layout";
+import { Switch, Redirect } from "react-router-dom";
+import { FormattedMessage as T } from "react-intl";
+import SecurityTab from "./SecurityPage/SecurityPage";
+import PrivacyTab from "./Privacy/Privacy";
+import { usePrivacyPage } from "./hooks";
+
+const PrivacyPageHeader = React.memo(
+  () => (
+    <StandaloneHeader
+    iconClassName="security"
+    title={<T id="privacypage.title" m="Privacy and Security Center" />}
+    description={
+      <T
+        id="privacy.description"
+        m="Various tools that help in different aspects of crypto currency privacy and security will be located here."
```

Comment:
```suggestion
        m="Tools that help in different aspects of cryptocurrency privacy and security."
```

Reply:
Thank you for your review. I've changed the text.